src/components/fields/select-field.vue

```diff
@@ -85,4 +104,19 @@ const selectedKey = computed({
     emit("update:modelValue", newValue);
   },
 });
+
+const filteredValues = computed(() => {
+  const query = searchQuery.value.toLowerCase();
+  return propsValue.value.filter((item) => getLabelFromValue(item).toLowerCase().includes(query));
+});
+
+watch(isDropdownOpen, async () => {
+  if (isDropdownOpen.value) {
+    await nextTick();
+    const el = document.querySelector("[data-radix-select-viewport]");
+    el.addEventListener("keydown", (event) => {
+      keydownHandler(event as KeyboardEvent);
+    });
+  }
+});
```

Comment:
What is this for?

Reply:
That watch for blocking filter in dropdown, when you press some letter, and it throws you to the first element with this letter